packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -4497,7 +4497,7 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
   }
 
   void _schedulePeriodicPostFrameCallbacks([Duration? duration]) {
-    if (!_hasInputConnection) {
+    if (!_hasInputConnection || !mounted) {
```

Comment:
I don't think this is going to change anything. if `mounted` is false then there won't be an input connection when the widget tree / render tree is finalized (see the `EditableTextState.dispose` implementation).

Reply:
ok will close this then to not pollute the area